lib/forwardable.rb

```diff
@@ -132,10 +132,10 @@ class << self
   #
   def instance_delegate(hash)
     hash.each do |methods, accessor|
-      unless defined?(methods.each)
-        def_instance_delegator(accessor, methods)
+      if methods.respond_to?(:each)
+        methods.each { |method| def_instance_delegator(accessor, method) }
       else
-        methods.each {|method| def_instance_delegator(accessor, method)}
+        def_instance_delegator(accessor, methods)
```

Comment:
Do you mind explaining this change in more detail? It seems like a nice improvement if we can avoid doing `each` and defer to some other method which already implements that logic.

Reply:
Just FYI, it seems like we can remove loop entirely. To me, that's a useful improvement, if it was/is redundant.